IapExample/android/app/build.gradle

```diff
@@ -162,6 +162,17 @@ android {
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
     }
+    flavorDimensions "appstore"
```

Comment:
An example where the dimension name in the client app is different from the one in the library